clang/lib/Sema/AnalysisBasedWarnings.cpp

```diff
@@ -544,25 +544,17 @@ static ControlFlowKind CheckFallThrough(AnalysisDeclContext &AC) {
 namespace {
 
 struct CheckFallThroughDiagnostics {
-  unsigned diag_MaybeFallThrough_HasNoReturn;
-  unsigned diag_MaybeFallThrough_ReturnsNonVoid;
-  unsigned diag_AlwaysFallThrough_HasNoReturn;
-  unsigned diag_AlwaysFallThrough_ReturnsNonVoid;
+  unsigned diag_FallThrough_HasNoReturn;
+  unsigned diag_FallThrough_ReturnsNonVoid;
   unsigned diag_NeverFallThroughOrReturn;
-  enum { Function, Block, Lambda, Coroutine } funMode;
+  unsigned funMode; // TODO: use diag::FunModes
```

Comment:
```suggestion
  unsigned FunMode; // TODO: use diag::FunModes
```
nit

Reply:
Are you sure? The first letter of each variable is in lower case...

Reply:
The current naming convention for variables is PascalCase; this here is just old code...

Reply:
Ah.. I see. Will fix :)